lisa/sut_orchestrator/azure/features.py

```diff
@@ -783,3 +787,116 @@ def get_azure_disk_type(disk_type: schema.DiskType) -> str:
     schema.DiskType.StandardHDDLRS: "Standard_LRS",
     schema.DiskType.StandardSSDLRS: "StandardSSD_LRS",
 }
+
+
+class Resize(AzureFeatureMixin, features.Resize):
+    def resize(self, resize_action: ResizeAction = None) -> schema.NodeSpace:
```

Comment:
@squirrelsc I thought it would be better to make the resize feature default so that the resize could increase or decrease the core count instead of having a default value that only allows either increase or decrease. Do you think the feature should be implemented in this way? If so, do you think that leaving the resize_action variable value as "None" is the best way to do so or should I add a new value to the ResizeAction enum?

Reply:
It should use a default value to increase_core_count. In a normal case, it doesn't need to specify the value, unless we need to test decrease the core count. The None doesn't bring value to the test case, in my mind.

Reply:
Done